src/lib/descriptors/Light/LightDescriptorBase.js

```diff
@@ -188,10 +188,10 @@ class LightDescriptorBase extends Object3DDescriptor {
     }
   }
 
-  hasColor() {
+  hasColor(colorType = 'color') {
```

Comment:
late again (i shouldn't promise days for delivery) but it's fine, I use it for material descriptor too :)